packs/hubot/rules/notify_hubot.yaml

```diff
@@ -7,7 +7,7 @@ trigger:
   pack: "chatops"
   type: "core.st2.generic.notifytrigger"
 criteria:
-  trigger.channel:
+  trigger.routes:
```

Comment:
I need to check the code, but shouldn't this be `route`, not `routes`?

If routes is an array / list then equal operator won't work on it.